ui/vty/src/Ariadne/UI/Vty/Widget/Account.hs

```diff
@@ -4,9 +4,10 @@ module Ariadne.UI.Vty.Widget.Account
 
 import Universum
 
+import Control.Exception.Base (handle)
```

Comment:
Usually `Control.Exception` is imported, not `Control.Exception.Base`. To be honest, I don't know what is `Control.Exception.Base` and how it's different from `Control.Exception` :man_shrugging: 
I don't insist on using `Control.Exception`, just find `Control.Exception.Base` a bit strange.

Reply:
Dunno, I hoogle'd it and didn't think any more